### ISSUE TITLE
useCheckoutPricing hook typings

### ISSUE THREAD

**Issue by ashokdudhade-cbs** [recurly/react-recurly#72]
We have a usecase to handle checkout pricing errors, with current typings I can't pass 2nd parameter(`setRecurlyError`) to the hook as mentioned in below example.

https://recurly.github.io/react-recurly/?path=/docs/hooks-usecheckoutpricing--page#error-handling

Can you update typings to support it?

**Comment by dbrudner** [comment]
Thanks for opening this! Good call, the type needs to be updated.

I opened this PR to fix the type https://github.com/recurly/react-recurly/pull/73.

A workaround for now would be to cast `useCheckoutPricing` as the correct type:

```
import { UseCheckoutPricingInput, UseCheckoutPricingReturn } from '@recurly/react-recurly';
import { RecurlyError } from '@recurly/recurly-js';
...

type UseCheckoutPricingWithErrorHandler = (
  input: UseCheckoutPricingInput,
  handleError?: (e: RecurlyError) => any
) => UseCheckoutPricingReturn;

const myCheckoutPricingInput = {};

const [
    { price, loading },
    setPricing
  ] = (useCheckoutPricing as UseCheckoutPricingWithErrorHandler)(myCheckoutPricingInput, (e: RecurlyError) => {});
...
```

Feel free to drop a review on the PR if you'd like. This should be fixed and released relatively soon 👍 .

**Comment by ashokdudhade-cbs** [comment]
@dbrudner  Thanks for the quick fix.

**Comment by dbrudner** [comment]
The type here has been fixed and released v1.2.2 https://github.com/recurly/react-recurly/releases/tag/v1.2.2

Thanks again for reporting!
